appdatabase/migrations/sql/1686048342_pubsubtopic_key.up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS pubsubtopic_key (
```

Comment:
Perhaps this could be named `pubsubtopic_signing_key`? Just to make clear the purpose of the key.